bdio-protobuf/src/main/proto/bdba_scan_nodes_v1.proto

```diff
@@ -0,0 +1,89 @@
+syntax = "proto3";
+
+package bdio.v1;
+
+import "google/protobuf/timestamp.proto";
+
+option java_multiple_files = true;
+option java_package = "com.blackducksoftware.bdio.proto.v1";
+
+
+enum MatchType {
+  SOME_MATCH_TYPE = 0;
+}
+
+
+/**
+ * See here for guidelines on avoiding breaking changes to message formats: https://developers.google.com/protocol-buffers/docs/proto#updating
+ * If those are not possible, or the message is sufficiently different, a new version should be created.
+ */
+message ProtoDependencyNode {
+  // the numeric id of component node
+  int64 depends_on = 1;
+
+  // the numeric id of file node
+  optional int64 evidence = 2;
+
+  // the id of layer node
+  optional string container_layer = 3;
+
+  // the id of layer node
+  optional string whiteout_layer = 4;
+
+  // description of this node
+  string description = 5;
+
+  // the list of match types (describes how evidence (evidence field) was matched to component (depends_on field)) 
+  repeated MatchType matchTypes = 6;
+}
+
+message ProtoComponentNode {
+  int64 id = 1;
+
+  // namespace of the component
+  string namespace = 2;
+
+  // identifier of the component
+  string identifier = 3;
+
+  // reference to annotation node
+  string description = 4;
+}
+
+message ProtoImageNode {
+  string id = 1;
+
+  string image_id = 2;
```

Comment:
These are coming from BDBA and shared across teams.   We should document what every field represents or means for shared understanding of production from the BDBA-side and consumption from the BD-side.

Reply:
Yes, this will be done after the meeting with BDBA team.